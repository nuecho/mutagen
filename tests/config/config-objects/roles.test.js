cfgObjectTests("roles", "roles-config.json", {checkMandatoryProperties: false});
