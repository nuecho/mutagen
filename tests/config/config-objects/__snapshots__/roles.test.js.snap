// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[mutagen roles config import-export] should create the roles: the roles have been created 1`] = `
Object {
  "stderr": "",
  "stdout": "+ role [cfgObjectTest]
Completed. 1 object created. 0 object updated. 0 object skipped.
",
}
`;

exports[`[mutagen roles config import-export] should skip the same roles: the roles have not been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "= role [cfgObjectTest]
Completed. 0 object created. 0 object updated. 1 object skipped.
",
}
`;

exports[`[mutagen roles config import-export] should update the modified roles: the modified roles have been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "~ role [cfgObjectTest]
Completed. 0 object created. 1 object updated. 0 object skipped.
",
}
`;
